docs/case-studies/buoyant-cloud.md

```diff
@@ -0,0 +1,83 @@
+---
+title: "Buoyant Cloud and Cortex: Standing on the shoulders of Linkerd and Prometheus"
+linkTitle: "Buoyant Cloud"
+weight: 3
+slug: buoyant-cloud
+---
+
+[Buoyant](https://buoyant.io), the creator of [Linkerd](https://linkerd.io), has had a close relationship with Prometheus and related technologies for years. As of today, that relationship now includes [Cortex](https://cortexmetrics.io/).
+
+Linkerd is an open source service mesh for Kubernetes, and part of the CNCF, along with Prometheus and Cortex. Linkerd provides three key classes of features: observability, reliability, and security—all without requiring any changes to your code. That first pillar, observability, has motivated deep Prometheus integration, and is the focus of this case study.
+
+## Linkerd x Prometheus
+
+In 2016, Linkerd 1.0 shipped with a [visualization module](https://linkerd.io/2016/10/04/a-service-mesh-for-kubernetes-part-i-top-line-service-metrics/#step-3-install-linkerd-viz) built with Prometheus and Grafana. With the release of [Linkerd 2.0 in 2018](https://linkerd.io/2018/09/18/announcing-linkerd-2-0/), Prometheus and Grafana were promoted to first-class components. That evolution was documented in a blog post entitled [Prometheus the Right Way](https://linkerd.io/2018/05/17/prometheus-the-right-way-lessons-learned-evolving-conduits-prometheus-integration/), and a KubeCon North America 2018 talk entitled [Linkerd 2.0, Now with Extra Prometheus](https://www.youtube.com/watch?v=bnDWApsH36Y), both co-authored by [Frederic Branczyk](https://twitter.com/fredbrancz) of Polar Signals and [Andrew Seigner](https://twitter.com/siggy) of Buoyant.
+
+This deep integration in Linkerd 2.0 provided out-of-the-box golden metrics, that is, success rate, request rate, and latency, across all your Kubernetes workloads.
+
+![Linkerd Grafana](/images/case-studies/bcloud-linkerd-grafana.png)
+_Linkerd 2.0 with integrated Prometheus and Grafana, out-of-the-box golden metrics_
+
+## Buoyant x Cortex
```

Comment:
I'm not sure what "Buoyant x Cortex" means. Is that multiplication?

Reply:
It is, I meant to imply "combining these two things is greater than the sum of the parts". =)